docker-compose.yaml

```diff
@@ -581,11 +581,63 @@ services:
       governance-multitenant-web:
         condition: service_started
 
+  nats-1:
+    image: nats:alpine
+    ports:
+      - "8222:8222"
+      - "4222:4222"
+    command: "-c /etc/nats/nats.conf --server_name nats-1"
+    volumes:
+      - nats-data-1:/nats-data
+      - ./resources/nats-local.conf:/etc/nats/nats.conf
+    networks:
+      - nats
+    labels:
+      app: nats
+      tier: infra
+    healthcheck:
+      test: ["CMD-SHELL", 'wget -qO- localhost:8222/healthz | grep ''"status":"ok"'' || exit 1']
+      start_period: 30s
+      interval: 10s
+      timeout: 10s
+      retries: 5
+  nats-2:
+    image: nats
+    command: "-c /etc/nats/nats.conf --server_name nats-2"
+    volumes:
+      - nats-data-2:/nats-data
+      - ./resources/nats-local.conf:/etc/nats/nats.conf
+    networks:
+      - nats
+    depends_on:
+      nats-1:
+        condition: service_started
+    labels:
+      app: nats
+      tier: infra
+  nats-3:
+    image: nats
+    command: "-c /etc/nats/nats.conf --server_name nats-3"
+    volumes:
+      - nats-data-3:/nats-data
+      - ./resources/nats-local.conf:/etc/nats/nats.conf
+    networks:
+      - nats
```

Comment:
We should make sure that the NATS servers are on the same Compose network as whatever needs to talk to it 🤔

Reply:
Yes, will add to the relevant ones when we integrate

Reply:
It will be the webhooks service: `governance-webhook-web`

Reply:
@henrymsiska @rblaine95 any other relevant changes to include? Can we approve / merge?

Reply:
@ff137 @rblaine95 : Will add the network to the correct component that will pull from redis to nats once that is added.